uber/templates/preregistration/preregbase.html

```diff
@@ -1,6 +1,8 @@
 {% extends "base.html" %}
 {% block head_javascript %}
     {{ super() }}
+
+    <script type="text/javascript">{% include "region_opts.html" %}</script>
```

Comment:
Every preregistration page is gonna get these opts whether the page needs it or not. I don't think thats TOO big of a deal, because `region_opts.html` is actually pretty small.